api/http/handler/stacks/handler.go

```diff
@@ -54,6 +54,8 @@ func NewHandler(bouncer *security.RequestBouncer) *Handler {
 		bouncer.AuthenticatedAccess(httperror.LoggerHandler(h.stackDelete))).Methods(http.MethodDelete)
 	h.Handle("/stacks/{id}",
 		bouncer.AuthenticatedAccess(httperror.LoggerHandler(h.stackUpdate))).Methods(http.MethodPut)
+	h.Handle("/stacks/{id}/git/config",
```

Comment:
@DimaSalakhov as per Chaim, I changed the handler router

Reply:
Either that or we can use something like `/stacks/{id}/redeploy` for the pull and redeploy instead.